scripts/extensions/pool_update.precheck

```diff
@@ -75,7 +75,9 @@ def parse_control_package(session, yum_url):
 def execute_precheck(session, control_package, yum_conf_file):
     if not control_package:
         return 'ok'
-    livepatch_dic = {'PATCH_PRECHECK_LIVEPATCH_COMPLETE': 'ok_livepatch_complete', 'PATCH_PRECHECK_LIVEPATCH_INCOMPLETE': 'ok_livepatch_incomplete', 'PATCH_PRECHECK_LIVEPATCH_NOT_APPLICABLE': 'ok'}
+    livepatch_dic = {'PATCH_PRECHECK_LIVEPATCH_COMPLETE': 'ok_livepatch_complete',
```

Comment:
Thanks @lindig. I agree `msg` is clearer than `key` in the traverse below. For `livepatch_dic`, since the messages only return from livepatch package, for other packages they will not return such messages, I think `livepatch_messages` is more appropriate than `yum_messages`.